content/vendor/guides/existing-cluster/_index.md

```diff
@@ -8,11 +8,67 @@ weight: "3"
+
+![Default YAML](/images/guides/kots/default-yaml.png)
+
+The default YAML documents above the white line contain information for Replicated, preflight checks, customer configuration screen options, and support bundle analyzers for troubleshooting installs. You can learn about those [in the reference docs](/reference/v1beta1) but for now let's click the "Save release" button in the bottom right.
+
+### Save and Promote Release
+
+Once the release is saved, go ahead and promote it to the Unstable channel to make this release available for installation. To do this, click the "Releases" link in the top left and then click the Promote button on the row we just created. In this popup, choose the Unstable channel and click the Promote button.
+
+![Create Application](/images/guides/kots/promote-release.png)
+
+Now that we've got a release promoted, we can walk through creating a license and installing our basic nginx application on a test server.
+
+* * *
+
+## Installing and Testing 
+
+This guide will give you first-hand experience installing a KOTS application using an existing Kubernetes cluster. If you haven't yet created a release, head back to the [Create and Promote a Release](../create-release) guide and complete that first.
+
+Now that we've created a release and promoted it to the Unstable channel, the next step is to create a customer license and use this this license to install the application on a test server.
+
+### Create License
+
+A customer license (downloadable as a `.yaml` file) is required to install any KOTS application. To create a customer license, log in to the [Vendor Portal](https://vendor.replicated.com) and select the Customers link on the left. Click the "Create a customer" button to continue.
+
+![Customers](/images/guides/kots/customers.png)
+
+On the Create a new customer page, fill in your name for the Customer name field, select the Unstable channel on the right hand side, and click Create customer. The defaults in all other fields will be fine.
+
+![Create Customer](/images/guides/kots/create-customer.png)
+
+After creating the customer, click the "Download license" link in the upper right corner. This will download the file with your customer name and a `.yaml` extension. This is the license file your customer will need to install your application. When a customer is installing your software you need to send them two things: the KOTS install script and the license file.
+
+### Create Kubernetes Cluster and Install Replicated
+
+KOTS can be installed either into an existing Kubernetes cluster or as an embedded cluster. You can see the installation options at the bottom of each channel on the Channels page.
 
-### Create Kubernetes Cluster
+![Installation Methods](/images/guides/kots/installation-methods.png)
+
+Installing KOTS on Existing cluster is very similar to Embedded cluster, except instead of bringing a plain VM we will use a pre-built Kubernetes cluster and deploy your KOTS app on top of it.
```

Comment:
```suggestion
Installing KOTS on Existing cluster is very similar to [using an embedded cluster](../quickstart#installing-and-testing), except instead of bringing a plain VM we will use a pre-built Kubernetes cluster and deploy your KOTS into a namespace.
```